app-frontend/src/app/components/leafletMap/leafletMap.controller.js

```diff
@@ -24,17 +60,21 @@ export default class LeafletMapController {
                 '</i> Find places</button>';
             return div;
         };
-        let zoom = L.control.zoom({position: 'topright'});
-        cartoPositron.addTo(map);
-        commandCenter.addTo(map);
-        zoom.addTo(map);
 
-        let $zoom = $('.leaflet-control-zoom');
-        let $mpc = $('.map-control-panel');
-        $($mpc).prepend($zoom);
+        cartoPositron.addTo(this.map);
+
+        if (!this.static) {
+            let zoom = L.control.zoom({position: 'topright'});
+            commandCenter.addTo(this.map);
+            zoom.addTo(this.map);
+
+            let $zoom = $('.leaflet-control-zoom');
```

Comment:
Using class names worries me a bit as it may break things if we ever have two maps on a page. Would be good to select it more explicitly if possible.